client/my-sites/customer-home/main.jsx

```diff
@@ -84,7 +84,7 @@ const Home = ( {
 				) }
 			</div>
 			<Notices checklistMode={ checklistMode } />
-			<Upsells />
+			{ layout && <Upsells cards={ layout.upsells } /> }
```

Comment:
Once these are all in, we should be able to move this truthy `layout` check higher up, since everything will rely on it.